modules/test/playwright/tests/workspaces/liferay-customer-workspace/pages/LoginPage.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+export class LoginPage {
```

Comment:
Do the Liferay Portal tests have any tests that do a login this way? Or is this login specific to our site?